### PR TITLE
Chore/fix lint warnings

### PR DESCRIPTION
**Pull request by nickvisut**
Fix lint warnings

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -30,11 +30,11 @@ export const compassTheme = createTheme({
     "0px 1px 3px 1px rgba(0, 0, 0, 0.15), 0px 1px 2px 0px rgba(0, 0, 0, 0.30)",
     "0px 2px 6px 2px rgba(0, 0, 0, 0.15), 0px 1px 2px 0px rgba(0, 0, 0, 0.30)",
     "0px 1px 3px 0px rgba(0, 0, 0, 0.30), 0px 4px 8px 3px rgba(0, 0, 0, 0.15)",
-    "0px 2px 3px 0px rgba(0, 0, 0, 0.30), 0px 6px 10px 4px rgba(0, 0, 0, 0.15)",
+    "0px 2px 3px 0px rgba(141, 59, 59, 0.3), 0px 6px 10px 4px rgba(0, 0, 0, 0.15)",
```
Comment (mrabbitt) [inline src/theme.ts:33]: Why are the RGB values for the color changed here?

Comment (mrabbitt) [reply src/theme.ts:33]: All the other changes look good to me, and they resolve the warnings that `npm run lint` shows in the main branch. ✅

Comment (KCCPMG) [reply src/theme.ts:33]: Same question, otherwise looks good to me too.

**Review by KCCPMG — CHANGES_REQUESTED** [review]

File: src/theme.ts (hunk 1 above)
Comment (nickvisut) [reply src/theme.ts:33]: good catch, thank you! no idea. maybe a color picker misclick.

**Review by mrabbitt — APPROVED** [review]

**Review by KCCPMG — APPROVED** [review]

**Pull request merged**